packages/features/schedules/components/ScheduleListItem.tsx

```diff
@@ -58,12 +60,32 @@ export function ScheduleListItem({
             <p className="text-subtle mt-1">
               {schedule.availability
                 .filter((availability) => !!availability.days.length)
-                .map((availability) => (
-                  <Fragment key={availability.id}>
-                    {availabilityAsString(availability, {
-                      locale: i18n.language,
-                      hour12: displayOptions?.hour12,
-                    })}
+                .map((availability) =>
+                  availabilityAsString(availability, {
+                    locale: i18n.language,
+                    hour12: displayOptions?.hour12,
+                  })
+                )
+                // sort the availability strings as per user's weekstart (settings)
+                .sort((a, b) => {
+                  const weekNames = weekdayNames(
+                    i18n.language,
+                    ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
+                      displayOptions?.weekStart || "Sunday"
+                    ) as 0 | 1 | 2 | 3 | 4 | 5 | 6,
+                    "short"
+                  );
+                  const weekIndex = (day: string) => {
+                    for (let i = 0; i < weekNames.length; i++) {
+                      if (day.includes(weekNames[i])) return i;
+                    }
+                    return -1;
+                  };
+                  return weekIndex(a) - weekIndex(b);
```

Comment:
Hi,
Instead of repeating this logic in both ScheduleListItem.tsx and AvailabilitySetigs.tsx, we can move this to common util and reuse it in both files.
Thanks